pandas/tests/tools/test_to_datetime.py

```diff
@@ -1882,7 +1891,10 @@ def test_dataframe_mixed(self, cache):
     def test_dataframe_float(self, cache):
         # float
         df = DataFrame({"year": [2000, 2001], "month": [1.5, 1], "day": [1, 1]})
-        msg = "cannot assemble the datetimes: unconverted data remains: 1"
+        msg = (
+            r"^cannot assemble the datetimes: unconverted data remains at position "
```

Comment:
NBD but is the "r" necessary?

Reply:
Shouldn't be, it's just a habit to always use that in regular expressions